sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -303,6 +295,27 @@ export class MessageSession extends LinkEntity<Receiver> {
         logger.logError(error, this.logPrefix);
         throw error;
       }
+    }
+
+    return link;
+  }
+
+  /**
+   * Creates a new AMQP receiver under a new AMQP session.
+   */
+  private async _init(
+    opts: { abortSignal?: AbortSignalLike; timeoutInMs?: number } = {}
+  ): Promise<void> {
+    try {
+      const sessionOptions = this._createMessageSessionOptions(this.identifier, opts.timeoutInMs);
+      await this.initLink(sessionOptions, opts.abortSignal);
+
+      if (this.link == null) {
```

Comment:
nitpick: should the check be strictier?
```suggestion
      if (this.link === null) {
```

Reply:
changed to `if (!this.link)`